hawc/apps/epi/serializers.py

```diff
@@ -136,14 +189,24 @@ class Meta:
         fields = "__all__"
 
 
+class ExposureWriteSerializer(serializers.ModelSerializer):
+    central_tendencies = CentralTendencySerializer(many=True, read_only=True)
+    form_integration_class = forms.ExposureForm
```

Comment:
is form_integration_class used anywhere? if not, delete from here and elsewhere

Reply:
yup sorry, remnant of FormIntegrationMixin. killed all these.